app/index.html

```diff
@@ -3,183 +3,12 @@
   <head>
     <title>311 calls</title>
     <link rel="stylesheet" href="//openlayers.org/en/v4.0.1/css/ol.css" type="text/css">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet"> /*Acess from google*/
```

Comment:
The page displays the text `/*Acess from google*/` at the top, which I believe is an error. Please fix. If that needs to be there, you can add it as an HTML comment: `<!-- Access from Google -->`.